### ISSUE TITLE
From which images is the FID computed?

### ISSUE THREAD

**Issue by kplum** [NVIDIA/pix2pixHD#250]
In the paper, did you for Cityscapes compute the FID only from the 500 image test set, or did you also consider original images from the training set in the computation of the statistics?
